src/cli.rs

```diff
@@ -54,6 +54,12 @@ use crate::normal_path::NormalPath;
     override_usage = "ghciwatch [--command SHELL_COMMAND] [--watch PATH] [OPTIONS ...]"
 )]
 pub struct Opts {
+    /// A Haskell source file to load into a `ghci` REPL.
+    ///
+    /// Shortcut for `--command 'ghci PATH'`. Conflicts with `--command`.
+    #[arg(value_name = "FILE", conflicts_with = "command")]
+    pub file: Option<NormalPath>,
```

Comment:
Could we move this after the `command` field here so it renders after `--command` in the `--help` message?